examples/service-clients/azure-client/external-controller/src/app.ts

```diff
@@ -175,10 +182,14 @@ async function start(): Promise<void> {
 	const diceRollerController1 = new DiceRollerController(diceRollerController1Props);
 	const diceRollerController2 = new DiceRollerController(diceRollerController2Props);
 
-	const contentDiv = document.getElementById("content") as HTMLDivElement;
+	const contentDiv = document.querySelector("#content") as HTMLDivElement;
 	contentDiv.append(
 		makeAppView([diceRollerController1, diceRollerController2], services.audience),
 	);
 }
 
-start().catch(console.error);
+try {
+	await start();
+} catch (error) {
+	console.error(error);
+}
```

Comment:
Here can just await start(), no need for try/catch.  The .catch() was just working around not being able to top-level await in non-module code and then getting hit by the floating promise lint rule.

Reply:
Good point. I'll double check that the linter configuration is okay with that for top-level awaits. It may not be. In that case, we may need to tweak the base config to support them.

Reply:
Actually, given that, do we even need a `start` function? Or should we just inline all of its contents?

Reply:
Good point, could do that.  I'm happy either way I think, probably just go with what seems nice aesthetically.

Reply:
I'll leave things as is (with a start function) for now. If nothing else, it offers scoping for variable names and such.